src/components/ui/OakRoundIcon/OakRoundIcon.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+import {
+  OakBox,
+  OakIcon,
+  OakIconProps,
+  OakImage,
+  OakImageProps,
+} from "@/components/base";
+
+type ImageProps = OakImageProps & {
+  iconName?: undefined;
+};
+
+type IconProps = OakIconProps & {
+  src?: undefined;
+};
+
+export type OakRoundIconProps = IconProps | ImageProps;
+
+/**
+ * A wrapper around `OakIcon` which applies a rounded background.
+ * Supports either an icon name or an image src
+ */
+export const OakRoundIcon = ({
```

Comment:
I thought `OakRoundIcon` was more descriptive than `OakContainedIcon`, happy to be challenged though!